bin/cachito-download.sh

```diff
@@ -80,17 +80,15 @@ inject_config_files () {
     curl -fsS "$request_url/configuration-files" > "$config_json"
 
     echo "Injecting configuration files to remote-source/"
-    # According to shellcheck, this is the proper way to save lines in an array
-    mapfile -t paths < <(jq '.[].path' -r < "$config_json")
-
-    for path in "${paths[@]}"; do
+    while IFS= read -r path; do
         # Show the path indented by 4 spaces
         echo "    $path"
         mkdir -p "$(dirname "$output_dir/remote-source/$path")"
 
         jq '.[] | select(.path == "'"$path"'") | .content' -r < "$config_json" |
             base64 --decode > "$output_dir/remote-source/$path"
-    done
+    done < <(jq '.[].path' -r < "$config_json")
```

Comment:
Personal preference: could this be `jq ... | while IFS= read ...` ?

I feel like it might be clearer than having the input data at the end of the while loop